linalgo/annotate/serializers.py

```diff
@@ -63,6 +64,14 @@ def _serialize(target):
         return s
 
 
+class BodySerializer:
+    @staticmethod
+    def _serialize(body):
+        if isinstance(body, str):
+            return {"body": body}
+        return asdict(body)
```

Comment:
`asdict` will only work on a dataclass. I don't think we can assume the user is providing a dataclass here.